### DISCUSSION TITLE
Inventory Heredity / Grouping

### DISCUSSION THREAD

**Discussion by MSandro** [semaphoreui/semaphore#1613] [Ideas]
Maybe its already possible, but if not, it would be great to allow inventorie referencing.
To describe this:
I have 8 inventories and I want an inventory called "All".
Of course, I don't want to have to configure everything twice. It would be better if I could just reference other inventories.
So the inventory would then have 8 child items.
Instead of having to write all the IPs of the other inventories in there again.

I am using the Static Inventory Type.

**Comment by hmoffatt** [comment]
Can't you do this within Ansible itself?

**Comment by MSandro** [comment]
I'm currently gaining my first experiences with Ansible. I've been working on automation for my company for two weeks. More and more servers are being included. At first glance, Semaphore appeared to be a good graphical interface so that people who don't know much about CLI tools can get an understanding of what is currently being built.

I know in Semaphore I can add additional CLI arguments (like multiple -i ) to a task template.
Another option would be to move the inventory to the Git repository and reference it as files. Ansible can also search folders. I'll try out how Semaphore handles this if I specify a folder instead of a file.
But is this really the correct approach?

**Comment by MSandro** [comment]
Okay, we are currently moving the inventories to the Git repository. It seems to work quite well with the folders. It currently only becomes difficult when different login data have to be used.

If there were the option to organize inventories into folders directly in Semaphore (or just by allowing to select childs in a inventory), it would certainly be much less complicated.

But for now, offloading to Git still seems to offer a small advantage.

**Comment by sickwell6988** [comment]
@MSandro @hmoffatt @benwyrosdick @tokuhirom @Elbandi 
Guys, I red carefully all the thread, and have the solution:

Just provide it like --limit and your_hosts. See below.

![image](https://github.com/user-attachments/assets/d5a11acc-e22f-4bd2-9da1-52ae5583766c)

Semaphore UI build: v2.12.14-685c726-1739570731
